crates/ruff/src/rules/tryceratops/rules/raise_vanilla_class.rs

```diff
@@ -7,6 +7,39 @@ use crate::registry::Diagnostic;
 use crate::violation::Violation;
 
 define_violation!(
+    /// ### What it does
+    /// Checks for bare exceptions.
+    ///
+    /// ## Why is this bad?
+    /// It's hard to capture generic exceptions making it hard for handling specific scenarios.
```

Comment:
FWIW, I am unable to understand these explanations. I would write something like this instead (disregarding `BaseException`'s existence for clarity):

```suggestion
    /// ### What it does
    /// Checks for code raises `Exception(...)` directly.
    ///
    /// ## Why is this bad?
    /// In order to handle such an exception, the handling code must use `except Exception`,
    /// which catches *any* raised exception, including failed assertions, division by zero,
    /// and others.
    ///
    /// Prefer to raise your own custom subclass of `Exception`. Then the handling code can
    /// catch only your specific exception type, without over-catching unintended
    /// exceptions.
```

Reply:
I just copied over from `Tryceratops` since I don't know what the repository stance is on these topics.

If I am allowed to write better explanations over using the existing ones I will do so!

Thanks for the help.